embed/templates/scripts/run_tidb.sh.tpl

```diff
@@ -17,7 +17,9 @@ cd "${DEPLOY_DIR}" || exit 1
   {{- end}}
 {{- end}}
 
-{{- if .NumaNode}}
+{{- if and .NumaNode .NumaCores}}
```

Comment:
Does specifying `NumaCores` without `NumaNode` make any sence?

Reply:
The user must specify NumaNode first, and then specify NumaCores，